<launch>
	<rosparam file="$(find tracking_controller)/cfg/controller_param.yaml" ns="controller"/>
	<node pkg="tracking_controller" type="tracking_controller_node" name="tracking_controller_node" output="screen" />
    
    <node name="rviz" pkg="rviz" type="rviz" output="screen"
        args="-d $(find sfc_planner)/rviz/sfc_navigation.rviz"> 
    </node>
    
    <node pkg="polys_mapgen" type="hpolys_mapgen_node" name="hpolys_mapgen_node" output="screen">
    <rosparam file="$(find polys_mapgen)/cfg/mapgen.yaml" command="load" />
    <remap from="/polys_pc_map" to="/voxel_map"/>
    </node>

	<node pkg="sfc_planner" type="sfc_navigation_node" name="sfc_navigation_node" output="screen">
    <rosparam file="$(find sfc_planner)/cfg/global_planning.yaml" command="load" />
    </node>
</launch>